src/detect_conflicts.py

```diff
@@ -174,5 +174,7 @@ def detect_conflicts(
     for mod in conflicts_by_mod:
         print(f"Conflicts with {mod}:")
         for mod2 in conflicts_by_mod[mod]:
-            print(f"- {mod2}: {conflicts_by_mod[mod][mod2]}")
+            to_print = f"- {mod2}: {conflicts_by_mod[mod][mod2]}"
+            print(to_print.encode('utf8'))  # fixes potential issue in windows terminal with failing to encode certain characters
```

Comment:
The output on my Windows is binary

```
Conflicts with Snow without patches:
b"- A Game of Thrones: ['snow_diffuse_for_combined_winter.dds', 'snow_diffuse.dds']"
```

Is adding `.decode()` after encode works on your system ?

Reply:
Hi! No, I could not find a clean way to fix that but ultimately decided that printing bytes wasn't a major concern.

That said I've whipped up an alternative fix by using io

```
import io
sys.stdout = io.TextIOWrapper(sys.stdout.detach(), sys.stdout.encoding, 'replace')
```

```
    for mod in conflicts_by_mod:
        print(f"Conflicts with {mod}:")
        for mod2 in conflicts_by_mod[mod]:
            print(f"- {mod2}: {conflicts_by_mod[mod][mod2]}")
        print("\n")
```

This appears to fix my issue while still printing strings correctly.